### ISSUE TITLE
Documentation for keyboard input needs work

### ISSUE THREAD

**Issue by aaronfranke** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#571]
**Describe the bug**

The documentation for keyboard input is confusing and needs work: https://github.com/OpenVR-Advanced-Settings/OpenVR-AdvancedSettings/blob/master/docs/keyboard_input_guide.md

Some things I am confused about after reading:

* The guide covers "Keyboard Shortcuts" and "Key Press", then in the below instructions it says to go to the section called `[keyboardShortcuts]`. So how does one set up a "Key Press"?

* There are zero examples of how to map buttons from controllers. For example, where is the table of how to get the up direction on the Index controller's left thumb stick?

* All of the examples are just tables with symbols, not code examples. It doesn't tell me what I'm supposed to put on the *left* of the equals sign. A better example would be how to bind a joystick to WASD, with the full code that goes in the INI.

* I already have some default data in this file. What does it mean? I mean, I see that there is a section that says "By Default Keyboard's are set to" but it doesn't actually convey information to me. What are these? What buttons on the Index/Vive/etc controllers would these Ctrl+Shift+M and F9 keys correspond to?

```
[keyboardShortcuts]
keyboardOne=^>m
keyboardTwo=^>m
keyboardThree=^>m
keyPressmisc=F9
keyPressSystem=F9
```

My goal is to play a game with my Index controllers that only natively supports keyboard/mouse + HMD. So I need a way to take buttons on my Index controllers and turn them into keyboard buttons. If someone could provide an example of how to map a joystick to WASD, that would be very much appreciated, and I might be able to figure out the rest myself.

**Software (please complete the following information):**
 - OS: Windows 10
 - Advanced Settings Version: Steam

**Comment by aaronfranke** [comment]
The OpenVR2Key program works great for my uses! Perhaps it's worth linking this tool in the documentation, something like "If you are trying to bind many keys, consider using OpenVR2Key instead of or in addition to OpenVR Advanced Settings".

**Comment by ykeara** [comment]
Fixed in 5.8.6 release
